### ISSUE TITLE
Including pretrained embeddings

### ISSUE THREAD

**Issue by ChristineBasta** [salesforce/awd-lstm-lm#75]
Hello,
I want to apply pre-trained embeddings of WordVec and Glove, I do not want them to be trained, so I commented in the model in forward function these lines,
  #emb = embedded_dropout(self.encoder, input, dropout=self.dropoute if self.training else 0)
   #emb = self.lockdrop(emb, self.dropouti)

Now the embeddings are returned as embedding object and the rest of the code needs it as tensor. So how to do this?? Is there something else I need to take care of in the rest of the code that can make a problem with the pretrained embeddings??
